scipy/optimize/_trustregion_constr/tests/test_report.py

```diff
@@ -13,6 +14,8 @@ def test_gh10880():
     minimize(lambda x: x**2, x0=2., method='trust-constr',
              bounds=bnds, options=opts)
 
+@pytest.mark.slow
+@pytest.mark.fail_slow(10)
```

Comment:
Not sure why, but this wasn't addressed in gh-20487. Seems fine to mark `xslow` to me; if so:
```suggestion
@pytest.mark.xslow
```

Reply:
Agreed, adding